src/utils/SessionValidation.js

```diff
@@ -0,0 +1,12 @@
+import LOG from './Log.js';
+
+const HandleResolverError = (session, error) => {
```

Comment:
Do we need to request logout from keycloak?

_Code from authRevoke.js_

![image](https://user-images.githubusercontent.com/36794274/199989733-d826ddd6-7ede-4a49-ae38-001708d0fe4b.png)